### DISCUSSION TITLE
How to add a missing bottle arch to existing formula?

### DISCUSSION THREAD

**Discussion by trallnag** [Homebrew/discussions#3606] [Writing Formulae/Casks] [answered]
The [gopass formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/gopass.rb) currently has the following bottles:

```
bottle do
  sha256 cellar: :any_skip_relocation, arm64_monterey: "1052163e1e32938898a794abb1fe90ff4454f7dddcb8930b3d0c3f78bc2533c6"
  sha256 cellar: :any_skip_relocation, arm64_big_sur:  "b16b4fcabf382b8b424f9f15ff24f9d2fbea5b4a9edb85c9f637ea69a89e93d7"
  sha256 cellar: :any_skip_relocation, monterey:       "6abbdaa26aa3fefd2b111f7211978136d87f39f39879282049cbc5fa7f839dec"
  sha256 cellar: :any_skip_relocation, big_sur:        "934ec918fa0c62dd83ed7c22bad69556d9a554b7fad645b9bafe6100c9cd215f"
  sha256 cellar: :any_skip_relocation, catalina:       "33164cc6369b06bc50ea6dc5bac4005e9669c5d0a78d33107b359f2a87a8fd18"
end
```

An entry for `x86_64_linux` is missing.

What needs to be done so that we can have a bottle for Linux?

From what I understand this is something only the Homebrew maintainers can do? And that there is some automation in place?

https://github.com/gopasspw/gopass/issues/2325

**Comment by SMillerDev** [comment, marked as answer]
Make a pull request to fix the build on Linux. That's all.

**Comment by danielnachun** [comment]
Note that `gopass` is in our list of formula we haven't been able to bottle: https://github.com/Homebrew/homebrew-core/issues/86422.  In this particular case it's because it fails the test on Linux but not macOS.  If you can figure out how to fix this that would be much appreciated as we haven't been able to yet.  It's possible that either the test needs to be tweaked because it's doing something wrong, or that the formula was built incorrectly, but unfortunately it's hard to tell without knowing how the formula works.
